enterprise/server/backends/pebble_cache/pebble_cache_test.go

```diff
@@ -3146,12 +3148,10 @@ func TestSampling(t *testing.T) {
 			// kick in. The unencrypted test digest should be evicted.
 			clock.Advance(minEvictionAge - 1*time.Minute)
 
-			for i := 0; i < 60; i++ {
+			for range evictions {
```

Comment:
Is this functionally any different from modifying the existing loop to remove the termination condition? That wouldn't require modifying the code being tested.

Reply:
that works too - done